app/views/users/index.html.erb

```diff
@@ -0,0 +1,32 @@
+<body>
+  <h1 class="title">Ruby on Rails Blog</h1>
+  <div class="user">
+    <img src="goku.jpg" alt="Goku" class="user_photo">
+    <div class="user_name">
+      <p class="name">User Number One</p>
+      <p class="post">Number of posts: x</p>
+    </div>
+  </div>
+  <div class="user">
+    <img src="vegeta.jpg" alt="Vegeta" class="user_photo">
+    <div class="user_name">
+      <p class="name">User Number Two</p>
+      <p class="post">Number of posts: x</p>
+    </div>
+  </div>
+  <div class="user">
+    <img src="Bulma.jpg" alt="Goku" class="user_photo">
+    <div class="user_name">
+      <p class="name">User Number Three</p>
+      <p class="post">Number of posts: x</p>
+    </div>
+  </div>
+  <div class="user">
```

Comment:
- [ ] [OPTIONAL] It would be great if you add a link to the posts pages from the user, for example by clicking on the user name the user can go to the show page of the specific user, and when clicking on individual post it can redirect the user to the post page